content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -132,6 +132,11 @@ The general workflow of a device plugin includes the following steps:
 * The plugin registers itself with the kubelet through the Unix socket at host
   path `/var/lib/kubelet/device-plugins/kubelet.sock`.
 
+  {{< note >}}
```

Comment:
I'm thinking we could also move the workflow to an ordered list?

Reply:
Sure, done.